velox/functions/lib/LikeFunctionUtils.h

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include <velox/type/StringView.h>
+#include <velox/vector/TypeAliases.h>
+#include <string>
+
+namespace facebook::velox::functions {
+namespace likeutils {
```

Comment:
I suggest to drop this namespace as we don't use such fine grain namespaces in the codebase.

All these functions are used only in one file: Re2Functions.cpp. Any reason not to move them into that file and place into anonymous namespace?

Reply:
Thanks for the input, removed likeutils namespace. Would like to keep a separate file since these functions are not strictly Re2 related, also makes it easier to reuse in other functions if necessary. Please let me know if these should be moved to Re2Functions.cpp.

Reply:
XxxUtils name is not very descriptive. We should avoid these names as much as possible. These functions are very simple and are used only in Re2Functions.cpp. It would be best to move them there.

Reply:
Got it, thanks for the clarification. Removed these files.

Reply:
Thanks.